### PR TITLE
Define ssd storage class for GCE

### PR DESCRIPTION
**Pull request by pschiffe**
GCE provides two storage options - hdd and ssd. During the OpenShift
installation, the default storage class is set to hdd, what is fine, but ssd
storage is not defined. This PR creates the ssd storage class during the
installation on GCE provider.

Let me know if this is not the proper way how to introduce an additional
storage class.

### CODE REVIEW

**Comment by pschiffe** [comment]
@ingvagabund what do you think? Should this kind of code be moved directly to the `openshift_default_storage_class` role, or something else?

File: playbooks/openshift-hosted/private/openshift_default_storage_class.yml [hunk 1]
```diff
@@ -4,3 +4,9 @@
   roles:
   - role: openshift_default_storage_class
```
Comment (ingvagabund) [inline playbooks/openshift-hosted/private/openshift_default_storage_class.yml]: Does it make sense to skip running this ``openshift_default_storage_class`` like it is for the `gce` if the role is run again with different values?

Comment (pschiffe) [reply playbooks/openshift-hosted/private/openshift_default_storage_class.yml]: No, the first run creates default hdd storage class, the second run creates non default ssd storage class.

**Comment by ingvagabund** [comment]
/hold
until the master branch is again open for new code

**Comment by ingvagabund** [comment]
@sdodson any ETA when the branch is open again?

**Comment by sdodson** [comment]
It wouldn't be open for feature work until February 22nd afaik. We're supposed to be focused on resolving bugs only at this point.

**Comment by pschiffe** [comment]
@ingvagabund @sdodson the gcp ci test passed, and it seems that failed tests didn't fail b/c of this pr.

**Pull request closed without merging**